frontend/packages/schema-editor/src/components/SchemaTree/SchemaNode/ActionButtons/ActionButtons.tsx

```diff
@@ -35,8 +38,9 @@ export const ActionButtons = ({ pointer, className }: ActionButtonsProps) => {
       <ActionButton
         color='danger'
         icon={<TrashIcon />}
-        titleKey='general.delete'
+        titleKey={actionButtonTitleKey}
         onClick={deleteNode}
+        disabled={savableModel.hasReferringNodes(pointer)}
```

Comment:
Bra! Resultatet av `savableModel.hasReferringNodes(pointer)` kan godt plasseres i en variabel, så blir ikke funksjonen kjørt to ganger.